infra/multi_cluster_service.tf

```diff
@@ -29,7 +29,7 @@ resource "google_compute_global_address" "multi_cluster_ingress_ip_address" {
   address_type = "EXTERNAL"
   project      = var.project_id
   depends_on = [
-    module.enable_google_apis
+    module.enable_base_google_apis
```

Comment:
Good thought. It _is_ strange that resources inside `multi_cluster_service.tf` don't all depend on multi-cluster APIs.
But I'll keep things as is because:
* This is the static IP address that we attach to the [MultiClusterIngress resource of the frontend](https://github.com/GoogleCloudPlatform/terraform-ecommerce-microservices-on-gke/blob/66c32b1f15de928615ecd233994b7920f4f84a38/kubernetes_manifests_deployer/kubernetes_manifests/multi_cluster_ingress.yaml#L51).
* That's why I've kept this `google_compute_global_address` resource in `multi_cluster_service.tf`.
* `google_compute_global_address` only relies on `compute.googleapis.com` which is enabled automatically when `container.googleapis.com`.